Makefile

```diff
@@ -55,6 +55,13 @@ else
 	rm temp-test.in
 endif
 	make install-flexmeasures
+	make install-highs
+
+install-highs:
+# Need to install highs locally on macos
+	if [ -d "../HiGHS/" ] && [ "$(shell uname)" = "Darwin" ]; then \
```

Comment:
How does the `HiGHS` directory exist there in the first place?

Reply:
It is created in "On MacOS it is impossible to properly install highs via pip, so it should be built locally:" part in setup-and-guidelines.rst I could add all the commands here but building is a long step so we should probably skip it in tests

Reply:
But this is the Makefile, which is independent of the docs. So I would actually put that code here (and then make the docs shorter), and only do the building when some marker does not exist yet (like maybe the directory?)